WordPress/Classes/Utility/HelpshiftUtils.m

```diff
@@ -46,33 +51,43 @@ - (void)checkIfHelpshiftShouldBeEnabled
 {
     NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
     [defaults registerDefaults:@{UserDefaultsHelpshiftEnabled:@NO}];
-
+    
     BOOL userHasUsedHelpshift = [defaults boolForKey:UserDefaultsHelpshiftWasUsed];
-
+    
     if (userHasUsedHelpshift) {
         [defaults setBool:YES forKey:UserDefaultsHelpshiftEnabled];
         [defaults synchronize];
         return;
     }
-
-    if (MPTweakValue(@"Helpshift Enabled", NO)) {
-        DDLogInfo(@"Helpshift Enabled");
-
-        NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
-        [defaults setBool:YES forKey:UserDefaultsHelpshiftEnabled];
-        [defaults synchronize];
-
-        // if the Helpshift is enabled we want to refresh unread count, since the check happens with a delay
-        [HelpshiftUtils refreshUnreadNotificationCount];
+    
+    if (MPTweakValue(@"Helpshift Enabled", YES)) {
+        [self enableHelpshift];
     } else {
-        DDLogInfo(@"Helpshift Disabled");
-
-        NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
-        [defaults setBool:NO forKey:UserDefaultsHelpshiftEnabled];
-        [defaults synchronize];
+        [self disableHelpshift];
     }
 }
 
+- (void)enableHelpshift
+{
+    DDLogInfo(@"Helpshift Enabled");
+    
+    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
+    [defaults setBool:YES forKey:UserDefaultsHelpshiftEnabled];
+    [defaults synchronize];
+    
+    // if the Helpshift is enabled we want to refresh unread count, since the check happens with a delay
+    [HelpshiftUtils refreshUnreadNotificationCount];
+}
+
+- (void)disableHelpshift
+{
+    DDLogInfo(@"Helpshift Disabled");
+    
+    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
+    [defaults setBool:NO forKey:UserDefaultsHelpshiftEnabled];
```

Comment:
@astralbodies - I don't think so because we never want to disable Helpshift if a user has already used it. I suppose I could add a comment.